### DISCUSSION TITLE
SetAtom type not exported

### DISCUSSION THREAD

**Discussion by bu3alwa** [pmndrs/jotai#1801] [General]
Is there a reason why the SetAtom type is not being exported? If I want to create a helper function and pass in SetAtom I am not able to type my parameter.

**Comment by dai-shi** [comment]
We are opinionated about not exporting types. (Sometimes, we change our mind. #1794 

Now, in v2 `SetAtom` is any function. So, I think defining it on your end isn't too strange.
See how we do it here:
https://github.com/jotaijs/jotai-game/blob/f8c729246904daaebd9f92c2e25fae662b4e4c84/src/useTransientAtom.ts#L6
